### PR TITLE
Fixes #1617 - Description placeholder update

### LINKED ISSUE #1617: bug description placeholder cut off at small viewports
See:

screen shot 2017-06-27 at 1 50 02 pm

Maybe we can think of some shorter example text?

### PR DESCRIPTION
**Pull request by cch5ng**
content: #1617; fix placeholder

### CODE REVIEW

**Comment by cch5ng** [comment]
@zoepage R?

File: webcompat/templates/home-page/form.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
             </div>
             <span class="wc-Form-validation">
             {{ form.description(class_='wc-Form-item required',
-               placeholder='e.g. The video should have started after 3 sec.',
+               placeholder="e.g. The video doesn't play",
```
Comment (zoepage) [inline webcompat/templates/home-page/form.html]: Is there a reason for using `"` and not ` ' `? Just curious. :)

Otherwise it looks good.

Comment (cch5ng) [inline webcompat/templates/home-page/form.html]: @zoepage it appeared the convention was to use double quotes for strings now. cannot recall now whether this was an issue from the prettier tool or not. #1347

Comment (zoepage) [inline webcompat/templates/home-page/form.html]: SGTM! r+

**Pull request merged**